src/mongocxx/options/find_one_and_replace.hpp

```diff
@@ -19,9 +19,11 @@
 #include <cstdint>
 
 #include <bsoncxx/document/view.hpp>
-#include <mongocxx/options/find_one_and_modify.hpp>
 #include <bsoncxx/stdx/optional.hpp>
```

Comment:
That may be more of an oversight than a design choice. Can anyone think of a good reason not to include the bsoncxx/stdx headers inside the mongocxx/stdx,hpp header?